sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_operations.py

```diff
@@ -117,6 +121,27 @@ def begin_sync_keys(self, name: Optional[str] = None) -> LROPoller:
         workspace_name = self._check_workspace_name(name)
         return self._operation.begin_resync_keys(self._resource_group_name, workspace_name)
 
+    @experimental
+    @monitor_with_activity(logger, "Workspace.BeginProvisionNetwork", ActivityType.PUBLICAPI)
+    @distributed_trace
+    def begin_provision_network(
+        self,
+        workspace_name: Optional[str] = None,
+        spark_enabled: Optional[bool] = False,
```

Comment:
can we make this parameter naming consistent with REST API?

Reply:
done, updated to include_spark